root/medium/MediumTracklist.js

```diff
@@ -62,6 +62,7 @@ const MediumTracklist = ({
       <tr
         className={loopParity(index) + (track.editsPending ? ' mp' : '')}
         id={track.gid}
+        key={index}
```

Comment:
I think you'll need to add a `key` to the data track "tr" above, too, otherwise you'll get warnings in that case (I saw some in the console and assume that is the reason).